Tests/LibWeb/Text/input/clone-node-attributes.html

```diff
@@ -0,0 +1,26 @@
+<script src="../include.js"></script>
```

Comment:
This test file doesn't seem to have a corresponding `expected` file?

Reply:
Whoops, I must have accidentally commited this while rebasing with some other WIP changes. Dropped this test file.